Packs/Malwarebytes/ReleaseNotes/1_1_4.md

```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Malwarebytes
+- Fix for RTP Detections(EP) incident creation.
```

Comment:
Suggest giving a bit more detailed explanation so users who had the issue can know it has been fixed. See  this for  more info
https://xsoar.pan.dev/docs/documentation/release-notes#bug-fixes-examples

Reply:
@ShahafBenYakir yeah, changed it now.